modules/mvt/src/lib/binary-vector-tile/vector-tile-feature.d.ts

```diff
@@ -0,0 +1,16 @@
+import Protobuf from 'pbf';
+
+export default class VectorTileFeature {
```

Comment:
Nice to add short JSDoc on all classes and functions, in this case perhaps stating that this is mapbox pbf wrapper or something. It shows up in vscode when hovering, helps get oriented.

Reply:
Done